ui/src/components/song/SongEditorDesktopView.tsx

```diff
@@ -196,6 +196,11 @@ const SongEditorDesktopView: FC<SongEditorProps> = ({ actionOnEditor }) => {
                     register={register}
                     multiple
                     freeSolo
+                    getOptionDisabled={(option) =>
+                      themes?.length === themeSelectionLimit || themes?.includes(option)
+                        ? true
+                        : false
+                    }
```

Comment:
I think can add a dialog box abt how the max themes is 3. Might confuse ppl when the themes suddenly got disabled.

Reply:
<img width="358" alt="image" src="https://github.com/hmcc-global/psalted-2/assets/26876443/7d2211ce-7ed9-4d5e-aab6-3108be6612cf">
ok added